### DISCUSSION TITLE
Buttons not showing up on any theme

### DISCUSSION THREAD

**Discussion by TylerDrowning** [amnweb/yasb#73] [Q&A] [answered]
![zen_D6s7vA4vmV](https://github.com/user-attachments/assets/d9f0aaca-1594-44b8-9ebd-524a804c5b79)
![yasb_Hgbv1il9F5](https://github.com/user-attachments/assets/cbb172af-b4a6-4016-89db-e3236d472dcf)

I'm on Windows 11, and I've downloaded the JetBrainsMono fonts from NerdFonts. I've tried multiple themes and none of the buttons will show up.

**Comment by amnweb** [comment]
Can you reset this theme? Delete the yasb folder from `.config` and start YASB again to generate a new default config, and be sure that you have JetBrains Mono NFP installed. In the screenshot you showed, I can clearly see that it is a font problem, not loaded or not installed.

Also restart YASB when you install font, if still YASB wont load fonts try restart PC.

**Reply by TylerDrowning** [reply]
Which one is the NFP? I installed it from NerdFonts on Powershell. I tried the other steps though, and it's still doing the same thing. JetBrainsMono is in my fonts on windows as well. 
![ApplicationFrameHost_jux8F948xW](https://github.com/user-attachments/assets/e846ca93-6893-4944-b275-8ad9e1f8b6a5)

**Reply by TylerDrowning** [reply]
[yasb.log](https://github.com/user-attachments/files/17835892/yasb.log)
Here's the yasb.log too just in case that helps.

**Reply by amnweb** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/2e54bdfa-a39b-4ed5-86f6-60af8125f95c)
Download jetbrains zip folder from nerd fonts site and install all

**Comment by TylerDrowning** [comment]
Shit, thank you lol
